src/datadoc/frontend/callbacks/register_callbacks.py

```diff
@@ -178,3 +180,25 @@ def register_new_variables_tab_callbacks(app: Dash) -> None:
     ready for production.
     """
     logger.info("Registering callbacks for the new variable tab for %s", app.title)
+
+    @app.callback(
+        Output("accordion-wrapper", "children"),
+        Input("language-dropdown", "value"),
+        Input("open-button", "n_clicks"),
+        prevent_initial_call=True,
+    )
+    def callback_new_variables_list(value: str, n_clicks: int) -> list:  # noqa: ARG001
+        response = {}
+        respons_list: list = []
+        index = 0
+        if n_clicks and n_clicks > 0:
+            for v in state.metadata.meta.variables:
+                index += 1
+                response = get_display_values(v, state.current_metadata_language)
+                respons_list.append(
+                    build_ssb_accordion(
+                        response["short_name"],
+                        {"type": "variabels-accordion", "id": index},
```

Comment:
Typo, needs to be updated a few other places too

```suggestion
                        {"type": "variables-accordion", "id": index},
```